includes/api/class-edd-api.php

```diff
@@ -1589,8 +1589,8 @@ public function get_discounts( $discount = null ) {
 			$paged     = $this->get_paged();
 			$per_page  = $this->per_page();
 			$discounts = edd_get_discounts( array(
-				'posts_per_page' => $per_page,
-				'paged'          => $paged,
+				'number' => $per_page,
+				'offset' => $paged * $per_page,
```

Comment:
```suggestion
				'offset' => ( absint( $paged ) - 1 ) * $per_page,
```

Otherwise it's setting an offset on page 1.